src/plugins/formvalid/reports/a11y-1-fr.hbs

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Validation de formulaires",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```

Comment:
Make the same edit as proposed in the other pages

Reply:
done